cfme/intelligence/reports/reports.py

```diff
@@ -297,7 +223,59 @@ def _get_state():
             )
 
 
-class CustomSavedReport(Updateable, Pretty):
+@navigator.register(CustomReport, 'All')
+class All(CFMENavigateStep):
+    prerequisite = NavigateToAttribute('appliance', 'LoggedIn')
+
+    def step(self):
+        from cfme.web_ui.menu import nav
+        nav._nav_to_fn('Cloud Intel', 'Reports')(None)
+        accordion.tree("Reports", "All Reports")
+
+
+@navigator.register(CustomReport, 'New')
+class New(CFMENavigateStep):
+    prerequisite = NavigateToSibling('All')
+
+    def step(self):
+        cfg_btn("Add a new Report")
+
+
+@navigator.register(CustomReport, 'Details')
+class Details(CFMENavigateStep):
+    prerequisite = NavigateToSibling('All')
+
+    def step(self):
+        accordion.tree(
+            "Reports", "All Reports", "{} (All EVM Groups)".format(
+                store.current_appliance.get_yaml_config("vmdb")["server"]["company"]
```

Comment:
This needs to change, to
```python
self.obj.appliance.get_yaml_config
```
else it will end up potentially getting it for the wrong appliance.